addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -594,15 +594,17 @@ options.Class.include({
             return;
         }
 
+        const fixCSSVariable = (value) => value.replace(/var\(--([0-9]+)\)/g, (match, group) => `var(--#{${group}})`);
+
         switch (type) {
             case 'views':
                 await this._customizeWebsiteViews(widgetValue, params);
                 break;
             case 'variable':
-                await this._customizeWebsiteVariable(widgetValue, params);
+                await this._customizeWebsiteVariable(fixCSSVariable(widgetValue), params);
                 break;
             case 'color':
-                await this._customizeWebsiteColor(widgetValue, params);
+                await this._customizeWebsiteColor(fixCSSVariable(widgetValue), params);
```

Comment:
Wouldn't it be better python side? Or at least in `_makeSCSSCusto`, so that it's a single point of failure in case we change methods around? What do you think? 😄

Reply:
@detrouxdev Sorry for the delay! I had to check other stuff before this one...
Indeed, a general python solution will be better for future updates... I moved the fix to the `/website/make_scss_custo` controller now. Just let me know if something needs to be adjusted :+1: